addons/im_livechat/static/src/core/web/discuss_app_model_patch.js

```diff
@@ -4,26 +4,21 @@ import { Record } from "@mail/core/common/record";
 import { _t } from "@web/core/l10n/translation";
 import { patch } from "@web/core/utils/patch";
 
-patch(DiscussApp, {
-    new(data) {
-        const res = super.new(data);
-        res.livechat = {
-            extraClass: "o-mail-DiscussSidebarCategory-livechat",
-            id: "livechat",
-            name: _t("Livechat"),
-            hideWhenEmpty: true,
-            canView: false,
-            canAdd: false,
-            serverStateKey: "is_discuss_sidebar_category_livechat_open",
-            sequence: 20,
-        };
-        return res;
-    },
-});
-
 patch(DiscussApp.prototype, {
     setup(env) {
         super.setup(env);
-        this.livechat = Record.one("DiscussAppCategory");
+        this.livechatThreads = Record.many("Thread", { inverse: "appAsLivechatThreads" });
```

Comment:
Could be just named `livechats` (simpler, single word rather than 2, chat is included in thread so thread is redundant)